Ryujinx/Ui/MainWindow.cs

```diff
@@ -0,0 +1,564 @@
+
+        private MainWindow(Builder builder, string[] args, Application gtkapp) : base(builder.GetObject("_mainWin").Handle)
+        {
+            _renderer = new OglRenderer();
+
+            _audioOut = InitializeAudioEngine();
+
+            _device = new HLE.Switch(_renderer, _audioOut);
+
+            Configuration.Load(System.IO.Path.Combine(AppDomain.CurrentDomain.BaseDirectory, "Config.json"));
+            Configuration.InitialConfigure(_device);
+
+            ApplicationLibrary.Init(SwitchSettings.SwitchConfig.GameDirs, _device.System.KeySet, _device.System.State.DesiredTitleLanguage);
+
+            _gtkapp = gtkapp;
+
+            ApplyTheme();
+
+            if (DiscordIntegrationEnabled)
+            {
+                DiscordClient   = new DiscordRpcClient("568815339807309834");
+                DiscordPresence = new RichPresence
+                {
+                    Assets = new Assets
+                    {
+                        LargeImageKey  = "ryujinx",
+                        LargeImageText = "Ryujinx is an emulator for the Nintendo Switch"
+                    },
+                    Details    = "Main Menu",
+                    State      = "Idling",
+                    Timestamps = new Timestamps(DateTime.UtcNow)
+                };
+
+                DiscordClient.Initialize();
+                DiscordClient.SetPresence(DiscordPresence);
+            }
+
+            builder.Autoconnect(this);
+
+            DeleteEvent += Window_Close;
+
+            _mainWin.Icon            = new Gdk.Pixbuf(Assembly.GetExecutingAssembly(), "Ryujinx.Ui.assets.RyujinxIcon.png");
+            _nfc.Sensitive           = false;
+            _stopEmulation.Sensitive = false;
+
+            if (SwitchSettings.SwitchConfig.GuiColumns[0]) { _iconToggle.Active       = true; }
+            if (SwitchSettings.SwitchConfig.GuiColumns[1]) { _titleToggle.Active      = true; }
+            if (SwitchSettings.SwitchConfig.GuiColumns[2]) { _developerToggle.Active  = true; }
+            if (SwitchSettings.SwitchConfig.GuiColumns[3]) { _versionToggle.Active    = true; }
+            if (SwitchSettings.SwitchConfig.GuiColumns[4]) { _timePlayedToggle.Active = true; }
+            if (SwitchSettings.SwitchConfig.GuiColumns[5]) { _lastPlayedToggle.Active = true; }
+            if (SwitchSettings.SwitchConfig.GuiColumns[6]) { _fileExtToggle.Active    = true; }
+            if (SwitchSettings.SwitchConfig.GuiColumns[7]) { _fileSizeToggle.Active   = true; }
+            if (SwitchSettings.SwitchConfig.GuiColumns[8]) { _pathToggle.Active       = true; }
+
+            if (args.Length == 1)
+            {
+                // Temporary code section start, remove this section when game is rendered to the GLArea in the GUI
+                _box.Remove(_glScreen);
+                if (SwitchSettings.SwitchConfig.GuiColumns[0]) { _gameTable.AppendColumn("Icon"       , new CellRendererPixbuf(), "pixbuf", 0); }
```

Comment:
```suggestion
                if (SwitchSettings.SwitchConfig.GuiColumns[0]) { _gameTable.AppendColumn("Icon",        new CellRendererPixbuf(), "pixbuf", 0); }
```
The comma need to be at the end of the args. The alignment is after the comma.
Ditto on the next lines.